MapboxNavigation/NavigationMapView.swift

```diff
@@ -354,24 +363,32 @@ open class NavigationMapView: MGLMapView {
         }
     }
     
+    var activeRouteIndex = 0
+    
+    var routes: [Route]?
+    
     /**
      Adds or updates both the route line and the route line casing
      */
-    public func showRoute(_ route: Route, legIndex: Int? = nil) {
+    public func showRoutes(_ routes: [Route], activeRouteIndex: Int = 0, legIndex: Int = 0) {
```

Comment:
Assume that `routes` is ordered from front to back or back to front (your choice), so `activeRouteIndex` becomes redundant.

Reply:
I agree, but it more explicit to name it. We apply the same logic to legs (legIndex) and step (stepIndex) as well.

Reply:
When it comes to tracking the current step in the leg, we have no choice, because the order of the steps matters and we can’t rearrange them arbitrarily. The order of routes is unimportant and arbitrary, other than having the active one be first or last. (In fact, if not for the need to mark one of them as the active route, I would advocate making it a set of routes.)

As precedent, the map SDK orders style layers from front to back in drawing order.